contents/handbook/people/feedback.md

```diff
@@ -26,34 +26,19 @@ With us growing the team, we will be splitting the session into smaller groups i
 - That being said, avoid piling on and repeating feedback others have given unless you have a different perspective or can add more context. It is ok to say "+1 to what X said about Y" and move on. Do not spend 2min repeating the same point that has already been made by someone else. 
 - Everyone is responsible for noting down and actioning their own feedback (ie. the people team won't do this for you). 
 
-## Performance reviews
+## Career chats
 
-In addition to informal day-to-day feedback and the full team 360 degree feedback session, it is important that we enable team members to take a step back every so often and look at their performance and aspirations in a wider context. This helps us to support a team member's growth and ensure it is aligned with PostHog's needs. 
+It's difficult for managers to take a step back in weekly 1:1s and talk about long term career goals. The manager is responsible for making sure a calendar invite exists that repeats every six months as a reminder to do this.
 
-This process is intended to be self-serve. The People team will ensure the process is kicked off and recorded properly, but it is the individual team member's responsibility to run the process. If you need support, ask your manager, Coua or Charles for help. 
+The process is as follows:
+1. You (the managee) gather feedback from your peers about what you should do to grow and what you could improve on.
```

Comment:
Alternative suggestion: align these with off-sites, where we already get feedback from peers. Furthermore this gives us a possibility to have this chat in person with the manager instead of over zoom, which would be a lot nicer.

Reply:
That's a good suggestion! I think managers might struggle to find 5x1 hours during an offsite to have these chats though and they might not sync up, but maybe it's just about encouraging people to do this at an offsite if one happens to be close?

Reply:
Yeah, could be an optional thing - 'if it happens to fall near an offsite, you might want to schedule this in person'. But I don't think it should be mandatory.

Reply:
I think this point (1) can be encouraged on an even more consistent basis than just every 6 months. From observation, it doesn't seem like much direct feedback is freely being searched for/given outside of 360 feedback/moments of strong disagreement. Personally speaking, it's always difficult to give feedback in the exact moment the thought comes up especially constructive feedback as it can come off as aggressive/reactive. Obviously, waiting 3 months to give the feedback feels too delayed too. 

So, I think encouraging everyone to be practicing (1) biweekly/monthly in asking and giving feedback should be a key tenant of "working/succeeding at posthog". It's much easier to give feedback when the receiver is explicitly open to and searching for it. I think there are more and more missed opportunities where we can really be calling out both fantastic work/help that was given and areas of miscommunication and strain

Reply:
We can just phase this as there are two options and it's up to you and your manager to decide which will work for you.